pypeit/pypeit.py

```diff
@@ -295,33 +330,48 @@ def reduce_exposure(self, frames, bg_frames=[], std_outfile=None):
         Args:
             frame (:obj:`int`):
                 0-indexed row in :attr:`fitstbl` with the frame to
-                reduce
-            bgframes (:obj:`list`, optional):
-                List of frame indices for the background
+                reduce.
+            bg_frames (:obj:`list`, optional):
+                List of frame indices for the background.
             std_outfile (:obj:`str`, optional):
-                the name of a file with a previously PypeIt-reduced standard spectrum.
+                File with a previously reduced standard spectrum from
+                PypeIt.
 
         Returns:
-            dict: The dictionary containing the primary outputs of extraction
+            dict: The dictionary containing the primary outputs of
+            extraction.
+
         """
+
+        # TODO:
+        # - bg_frames should be None by default
+        # - change doc string to reflect that more than one frame can be
+        #   provided
+
         # if show is set, clear the ginga channels at the start of each new sci_ID
         if self.show:
+            # TODO: Put this in a try/except block?
             ginga.clear_all()
 
-        # Save the frame
+        # Save the frames
+        # TODO: Should these be set to self?  They're not used elsewhere
+        # in the class or in run pypeit.
         self.frames = frames
```

Comment:
probably should not be set to self.

Reply:
Okay, not longer saving `frames` or `bg_frames` to `self`.